gpt4all-chat/qml/ChatView.qml

```diff
@@ -1694,17 +1694,21 @@ Rectangle {
                         imageHeight: 20
                         visible: chatModel.count && !currentChat.isServer && currentChat.isModelLoaded && !currentChat.responseInProgress
                         onClicked: {
-                            var index = Math.max(0, chatModel.count - 1);
-                            var listElement = chatModel.get(index);
+                            var promptIndex = Math.max(0, chatModel.count - 2);
+                            var promptElement = chatModel.get(promptIndex);
+                            var responseIndex = Math.max(0, chatModel.count - 1);
+                            var responseElement = chatModel.get(responseIndex);
                             currentChat.regenerateResponse()
                             if (chatModel.count) {
```

Comment:
I think this would be simpler if we do nothing if `chatModel.count < 2`, since that wouldn't be well-defined anyway. Then you don't need Math.max at least.